server/apps/microservices/src/processors/metadata-extraction.processor.ts

```diff
@@ -144,84 +143,42 @@ export class MetadataExtractionProcessor {
   async extractExifInfo(job: Job<IExifExtractionProcessor>) {
     try {
       const { asset, fileName }: { asset: AssetEntity; fileName: string } = job.data;
-      const exifData = await exifr.parse(asset.originalPath, {
-        tiff: true,
-        ifd0: true as any,
-        ifd1: true,
-        exif: true,
-        gps: true,
-        interop: true,
-        xmp: true,
-        icc: true,
-        iptc: true,
-        jfif: true,
-        ihdr: true,
+      const exifData = await exiftool.read(asset.originalPath).catch(e => {
+        this.logger.debug(`The exifData parsing failed due to: ${e}`);
+        throw new Error(`Can not parse exif data from file ${asset.originalPath}`);
```

Comment:
Perhaps just catch the error and log it; throwing an error will result in the function terminating here, correct?

Reply:
Yeah, just noticed that and also pushed a fix for it.